sources/packages/backend/libs/sims-db/src/entities/student-loan-balances.model.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from "typeorm";
+import { ColumnNames, TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+import { numericTransformer } from "../transformers/numeric.transformer";
+import { Student } from "./student.model";
+
+/**
+ * Stores files information and content from NSLSC
+ * Loan Balance file.
+ */
+@Entity({ name: TableNames.StudentLoanBalances })
+export class StudentLoanBalances extends RecordDataModel {
```

Comment:
As you pointed out, in the other PR from where this was copied, I also agree that this should be singular.